packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -187,6 +137,7 @@ export class TeamsBotImpl {
     this.ctx = context;
     await this.config.restoreConfigFromContext(context);
     Logger.info(Messages.PreProvisioningBot);
+    // todo: check on Azure whether resources are provisioned, if true, skip provision
```

Comment:
do we still need this skip logic? IMO this will be handled by ARM template, please check with Chaoyi.